docs/topics/proc_deploying-the-health-check-booster-using-the-oc-cli-client.adoc

```diff
@@ -5,11 +5,12 @@
 // Rationale: This procedure is identical in all deployments.
 
 [id='deploying-the-health-check-booster-using-the-oc-cli-client_{context}']
-= Deploying the {name-mission-health-check} booster using the `oc` CLI client
+[id='deploying-the-health-check-example-using-the-oc-client_{context}']
+= Deploying the {name-mission-health-check} example using the `oc` CLI client
```

Comment:
Deploying the {name-mission-health-check} example using --> Deploying the {name-mission-health-check} example application using